apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QSchemaColumnsIT.java

```diff
@@ -253,9 +253,7 @@ public void columnUpdateBadKeyspace() {
             HttpStatus.SC_BAD_REQUEST);
     ApiError apiError = readJsonAs(response, ApiError.class);
     assertThat(apiError.code()).isEqualTo(HttpStatus.SC_BAD_REQUEST);
-    // 02-Sep-2022, tatu: Error message is bit misleading but it is what it is; verify:
-    assertThat(apiError.description())
-        .matches("Table 'table' not found.*keyspace.*" + badKeyspace + ".*");
```

Comment:
One bonus improvement is that we get better error checking for keyspace argument.